plugins/woocommerce/tests/api-core-tests/tests/settings/settings-crud.test.js

```diff
@@ -141,8 +141,8 @@ test.describe('Settings API tests: CRUD', () => {
 				expect.arrayContaining([
 					expect.objectContaining({
 						id: "email_cancelled_order",
-						label: "Cancelled order",
-						description: "Cancelled order emails are sent to chosen recipient(s) when orders have been marked cancelled (if they were previously processing or on-hold).",
+						label: "Canceled order",
+						description: "Canceled order emails are sent to chosen recipient(s) when orders have been marked cancelled (if they were previously processing or on-hold).",
```

Comment:
Thanks for the review, those probably don't need to be changed, I was just trying to make it consistent. I can revert those ones.

Reply:
Hey @matt-h We need to update this since it's causing the API test to fail and would appreciate it if you could revert those unneeded changes. Thanks 🙏 

https://github.com/woocommerce/woocommerce/actions/runs/3840387935/jobs/6539380835